docs/topics/images.rst

```diff
@@ -204,6 +204,29 @@ Therefore, if you'd added the field ``author`` to your AbstractImage in the abov
     The image property used for the ``src`` attribute is actually ``image.url``, not ``image.src``.
 
 
+Alternative HTML tags
+`````````````````````
```

Comment:
Sphinx is complaining about inconsistent title levels here, presumably because we use `^` instead of backticks to indicate subheadings in the JPEG section below.

In this case I'd suggest shuffling things around so that this is the same level as its neighbours, though:

* "The attrs shortcut" should be moved above this, because the use of the `tmp_photo` variable directly follows on from the example code in "More control over the img tag"
* "Alternative HTML tags" should then be another top-level section (underlined with dashes).